app/decorators/offence_decorator.rb

```diff
@@ -12,7 +19,7 @@ def mode_of_trial_reason_list
     return t('generic.not_available') if mode_of_trial_reasons.blank?
     return mode_of_trial_reasons unless mode_of_trial_reasons.is_a?(Enumerable)
 
-    safe_join(mode_of_trial_reason_descriptions, tag.br)
+    safe_join(mode_of_trial_reason_descriptions.reject(&:empty?), tag.br)
```

Comment:
will [compact](https://ruby-doc.org/core-2.4.1/Array.html#method-i-compact) work here instead of `empty?`

```suggestion
    safe_join(mode_of_trial_reason_descriptions.compact, tag.br)
```

Reply:
no it won't actually. you could use `nil` instead of empty string and then use compact

Reply:
👍 have amended to use nil/compact

Reply:
cool, i think nil is better because it can also be used as a `falsey`